cos_alerter/alerter.py

```diff
@@ -0,0 +1,53 @@
+import fcntl
+import json
+import tomllib
+
+import apprise
+
+
+class Config:
+    def __getitem__(self, key):
+        with open('/etc/cos-alerter.toml', 'rb') as f:
+            return tomllib.load(f)[key]
```

Comment:
It seems like there's not necessarily a great reason to use TOML for this instead of YAML, but just a comment.

Reply:
I think it is a matter of personal preference in this case :P But yes, I think we should probably use yaml to make it consistent with our other projects.